diff -Naur libdrm-2.4.64/xf86drm.h libdrm-2.4.64.tpg/xf86drm.h
--- libdrm-2.4.64/xf86drm.h	2015-08-17 14:08:11.000000000 +0000
+++ libdrm-2.4.64.tpg/xf86drm.h	2015-08-22 11:42:27.197137074 +0000
@@ -74,7 +74,7 @@
 /* Default /dev/dri directory permissions 0755 */
 #define DRM_DEV_DIRMODE	 	\
 	(S_IRUSR|S_IWUSR|S_IXUSR|S_IRGRP|S_IXGRP|S_IROTH|S_IXOTH)
-#define DRM_DEV_MODE	 (S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP)
+#define DRM_DEV_MODE	 (S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP|S_IROTH|S_IWOTH)
 
 #ifdef __OpenBSD__
 #define DRM_DIR_NAME  "/dev"
